src/constants.js

```diff
@@ -2,4 +2,4 @@ export const DOMAIN = 'accessible-colors.com';
 export const SITE_URL = `http://${DOMAIN}`;
 export const REPO = 'moroshko/accessible-colors';
 export const MIN_FONT_SIZE = 10;
-export const MAX_FONT_SIZE = 21;
+export const MAX_FONT_SIZE = 28;
```

Comment:
If you already touching max font size, do you mind deciding what to do with https://github.com/moroshko/accessible-colors/issues/63? (either make the change, or close the issue)